docs/docs/building-services/gramine.md

```diff
@@ -74,21 +98,32 @@ You can specify the files' content in the MarbleRun manifest:
 ...
 ```
 
-Note that Gramine also allows to store files encrypted on the host's file system. The so called `protected files` require to initialize the protected files key by writing it hex-encoded to the virtual `protected_files_key` device:
+Note that Gramine also allows to store files encrypted on the host's file system. The so encrypted filesystem requires to initialize an encrypted files key by writing it to the virtual `protected_files_key` device.
```

Comment:
```suggestion
Note that Gramine also allows to store files encrypted on the host's file system. This is done by providing one or multiple encryption key and defining a mount point as encrypted in combination with the key to be used.
```

Or something simpler, but `protected_files_key` is deprecated and does not need to be mentioned here for the "for debug purpose" case.

Reply:
Shortened the section by referring to the Gramine docs where possible

Reply:
Generally that's fine, though I would still keep the distinction between the "debug" use and the proper key deployment with MarbleRun. The `insecure` should already give a hint, but since it's important for security better be safe than sorry I guess. 

Also, one might wonder why we even advertise the insecure way. Keeping the distinction between "quick for debug" versus recommendation for production use cases also allows us to show the value MarbleRun provides here (seamless key provisioning).

But that's the last nit I have, don't worry :)

Reply:
I would prefer to keep our docs on Gramine as minimal as possible, since Gramine's manifest syntax can, and does, change from one release to another.
Users should not need our docs to learn about, or understand Gramine's features.
The fact that encryption keys should be provisioned at runtime by an external application is already mentioned in their docs, and I think they explain it quite well.

Reply:
Oh never mind, I actually didn't fully realize in the diff that you removed the insecure example from the docs.
Sorry, too early in the morning. That's fine then :)